algolia/recommendation/client.go

```diff
@@ -0,0 +1,54 @@
+package recommendation
+
+import (
+	"github.com/algolia/algoliasearch-client-go/v3/algolia/call"
+	"github.com/algolia/algoliasearch-client-go/v3/algolia/compression"
+	"github.com/algolia/algoliasearch-client-go/v3/algolia/region"
+	"github.com/algolia/algoliasearch-client-go/v3/algolia/transport"
+)
+
+// Client provides methods to interact with the Algolia Recommendation API.
+type Client struct {
+	appID     string
```

Comment:
`algolia/recommendation/client.go:12:2: `appID` is unused (structcheck)` you got an error from the linter here

Reply:
Fixed!